addons/featured-lists/userscript.js

```diff
@@ -0,0 +1,59 @@
+export default async function ({ addon, console }) {
+  let modSplashes;
+  changeRowsOrder();
+
+  addon.self.addEventListener("disabled", function () {
+    modSplashes = document.querySelectorAll(".mod-splash");
```

Comment:
Why not put this line above where the the variable is initialized and just it throughout the file?